### DISCUSSION TITLE
Représentation des flux et entités

### DISCUSSION THREAD

**Discussion by Blemaire-ssi** [dbarzin/mercator#790] [Ideas]
Bonjour,

Dans Mercator, les flux de la vue application peuvent être à destination d'un application, un service, une base de données ou un module applicatif.

Ne serait-il pas pertinent de rajouter la possibilité de lier un flux applicatif à une entité ? 

Nous avons plusieurs cas d'usage qui envoient des mails, SMS ou courriers à destination d'entités que nous souhaitons modéliser.

**Comment by dbarzin** [comment]
Je comprends ce besoin, ma première idée est de modéliser l'application qui reçoit le flux dans l'entité externe, mais je comprends que ce n'est pas l'idéal. Cependant, en ajoutant ce type de destination, j'ai le sentiment que l'on mélange les domaines applicatifs et écosystème. Qu'en penses-tu ?

**Comment by Blemaire-ssi** [comment]
Tu as raison, la solution qui semble la plus logique pour bien séparer les différentes couches applicatives et écosystème est de représenter l'application qui va recevoir l'information (mail ou SMS). 
Je cherchais à trouver une meilleure façon de faire cette représentation mais je ne pense pas qu'il y en ait une.
